modules/nw-ptp-introduction.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+
+[id="ptp-introduction_{context}"]
+= Introduction to PTP
+
+The Precision Time Protocol (PTP) is a protocol used to synchronize clocks in a network. When used in conjunction with hardware support, PTP is capable of sub-microsecond accuracy, which is far more accurate than Network Time Protocol (NTP).
+
+The linuxptp package includes the `ptp4l` and `phc2sys` programs for clock synchronization. The `ptp4l` program implements the PTP boundary clock and ordinary clock. The `ptp4l` program synchronizes the PTP hardware clock to the source clock with hardware time stamping and synchronizes the system clock to the source clock with software time stamping. The `phc2sys` program is only needed for hardware time stamping to synchronize the system clock to the PTP hardware clock on the network interface card (NIC).
+
+[id="ptp-elements_{context}"]
+== Elements of a PTP domain
+
+PTP included multiple nodes connected through the network, each representing a clock. The following type of clocks can be included in configurations:
```

Comment:
This sentence is confusing, also "included" is past tense. 

What about "PTP is used to synconize multiple nodes connected in a network, with clocks for each node."